cypress/fixtures/fireboltCalls/accessibility.json

```diff
@@ -10,215 +10,321 @@
     },
     "DISABLE_CLOSEDCAPTIONS":{
         "method": "manage_closedcaptions.setEnabled",
-		"params": "FALSE"
+        "params": "FALSE",
+        "validationJsonPath": "result",
+        "expected": "NULL"
     },
```

Comment:
why are we adding the **validationJsonPath** and **expected** values here ? The purpose of this ticket is to reuse the same key  for setter call and setter validation. And the code should have the capability to handle if no **validationJsonPath** and **expected** are given here

Reply:
reverted the changes